app/helpers/report_helper.rb

```diff
@@ -5,19 +5,40 @@
 #   the COPYRIGHT file.
 
 module ReportHelper
+  # rubocop:disable Rails/OutputSafety
   def report_content(report)
     case (item = report.item)
     when Post
-      raw t("report.post_label", content: link_to(post_message(item), post_path(item.id)))
+      raw t("report.post_label", content: link_to(truncated_post_message(item), post_path(item.id)))
     when Comment
       raw t("report.comment_label", data: link_to(
-        h(comment_message(item)),
+        h(truncated_comment_message(item)),
         post_path(item.post.id, anchor: item.guid)
       ))
     else
       t("report.not_found")
     end
   end
+  # rubocop:enable Rails/OutputSafety
+
+  def link_to_content(report)
+    case (item = report.item)
+    when Post
+      link_to("", post_path(item.id),
+              {title:  "View reported element",
```

Comment:
This text should probably be translated.

Also, either write "View reported post" here, or if you keep "element", it could used for both which I think is fine as well (so less duplication needed, and only the actual links needs to be different).